lblod/app-gelinkt-notuleren
# Fix issues with signing extracts

### Overview
~~Will~~ includes changes in prepublication-service and the frontend. ~~This is a draft until versions with those changes exist.~~

Update mu-cl-resources config for SignedResource and PublishedResource to include files. Only file or content will exist for any one resource but mu-cl-resources seems to handle fields not existing comfortably.

##### connected issues and PRs:
Fixes https://github.com/lblod/frontend-gelinkt-notuleren/issues/639
~~Waiting on https://github.com/lblod/notulen-prepublish-service/pull/108~~
~~And https://github.com/lblod/frontend-gelinkt-notuleren/pull/640~~

### Setup
~~Update the image for the `editor` service to be the build containing the now merged changes to the frontend:
`lblod/frontend-gelinkt-notuleren@sha256:01030927f54c34afdc3185cb3930efb490e95f3be96bea7880a657425524f3b2`~~

### How to test/reproduce
See [Jira ticket]([binnenland.atlassian.net/browse/GN-4769](https://binnenland.atlassian.net/browse/GN-4769)) for reproduction instructions for the crash. For the unreported broken detail screen, it's necessary to follow the same flow as that video but for an extract with a signature. Where the 'Sign' button was, it's now possible to delete a signature, do that:
![image](https://github.com/lblod/app-gelinkt-notuleren/assets/1323250/9f47c310-4796-441a-aacb-7de7e8daa128)
Then, go to the 'Publication actions' tab and click on the document name for the deleted signature:
![image](https://github.com/lblod/app-gelinkt-notuleren/assets/1323250/410cb955-dd82-47d0-a08c-430d03cca0a2)
The expectation is to see a copy of the extract. The broken version contains just an empty box:
![image](https://github.com/lblod/app-gelinkt-notuleren/assets/1323250/7099aef6-a32e-438e-bbbb-69ead2b8c7f8)

### Challenges/uncertainties
N/A

### Checks PR readiness
